rest/model/model.go

```diff
@@ -0,0 +1,8 @@
+package model
+
+// Model defines how to transform to and from an API model for a given
+// interface.
+type Model interface {
+	Import(interface{}) error
+	Export(interface{}) (interface{}, error)
```

Comment:
I think that export doesn't need to take an argument? or at least there should be some more documentation about how this would get used.